### DISCUSSION TITLE
Configure multiple sources of vulnerabilities

### DISCUSSION THREAD

**Discussion by jbruenner** [DependencyTrack/dependency-track#1741] [Q&A]
Hi,

I'm running DT with NVD feeds and github feeds. That's why I'm getting a lot of vulnerabilities twice. Is there any trick to get rid of the 
doubles by configuration? I can't surpress them manually, they are too many. Do I have to choose betweene one of the sources?

Thanks for your help
Jörg

**Comment by nscuro** [comment]
Hi Jörg, this will be addressed in the upcoming 4.6 release, see https://github.com/DependencyTrack/dependency-track/issues/1642
